src/pages/Home/Home.page.jsx

```diff
@@ -1,38 +1,34 @@
-import React, { useRef } from 'react';
-import { Link, useHistory } from 'react-router-dom';
+import React from 'react';
 
-import { useAuth } from '../../providers/Auth';
-import './Home.styles.css';
+import {
+  Title,
+  VideosContainer,
+  VideoCard,
+  VideoPreview,
+  VideoDescription,
+  VideoTitle,
+  VideoContent,
+} from './Home.styled';
+import mockData from './youtube-videos-mock.json';
 
 function HomePage() {
-  const history = useHistory();
-  const sectionRef = useRef(null);
-  const { authenticated, logout } = useAuth();
-
-  function deAuthenticate(event) {
-    event.preventDefault();
-    logout();
-    history.push('/');
-  }
-
   return (
-    <section className="homepage" ref={sectionRef}>
-      <h1>Hello stranger!</h1>
-      {authenticated ? (
-        <>
-          <h2>Good to have you back</h2>
-          <span>
-            <Link to="/" onClick={deAuthenticate}>
-              ← logout
-            </Link>
-            <span className="separator" />
-            <Link to="/secret">show me something cool →</Link>
-          </span>
-        </>
-      ) : (
-        <Link to="/login">let me in →</Link>
-      )}
-    </section>
+    <>
+      <Title>Mini-Challenge 1</Title>
+      <VideosContainer>
+        {mockData.items
+          .filter((item) => item.id.kind === 'youtube#video')
+          .map((item) => (
+            <VideoCard key={item.id.videoId}>
+              <VideoPreview src={item.snippet.thumbnails.medium.url} />
+              <VideoContent>
+                <VideoTitle>{item.snippet.title}</VideoTitle>
+                <VideoDescription>{item.snippet.description}</VideoDescription>
+              </VideoContent>
+            </VideoCard>
```

Comment:
The way you keep the `VideoCard` components separate helps with readability, you can go one step further and move all the things related to the `VideoCard` to a separate file, this way you can test the component easily

Reply:
OK Thanks!